iroh-net/src/netcheck/reportgen.rs

```diff
@@ -1316,7 +1325,6 @@ mod tests {
     //
     // TODO: Not sure what about IPv6 pings using sysctl.
     #[tokio::test]
-    #[cfg_attr(target_os = "windows", ignore = "flaky")]
```

Comment:
:tada: :cold_sweat:

Reply:
I made the windows all-features, stable workflow run three times and it passed. I have no idea how many runs we feel to be comfortable with this again, but hey it's something? new commits will be the fourth one. Let's see how it goes